### ISSUE TITLE
I lost my personal preset when I upgraded

### ISSUE THREAD

**Issue by tyeeman** [CineEncoder/CineEncoder#25]
Yes, I unpacked the zipfile into my current folder and lost my preset. Where are they stored and why won't the newer version read it?

Thanks! Keep up the good work here!

**Comment by CineEncoder** [comment]
Hello! Indeed, there is such a problem with the compatibility of presets for different versions of the program, this is due to the fact that the list of encoding modes and parameters has changed in different versions. The presets file is located in the Home/CineEncoder (for Linux) and User/CineEncoder (for Windows) folders, but an attempt to replace the file with another version will most likely end with the program crashing. Starting from version 3.5.3, I think there will be no significant changes in the parameters, but again until some new codec or parameter is added. It is almost impossible to make the parameters file compatible with all versions (my opinion).

**Comment by tyeeman** [comment]
So are you saying starting in 3.5.3 and future versions, custom made presets will be stored in the same place all the time and will be read just fine? My preference is to store them in a "preferences" subfolder within Cine-Encoder's folder.

**Comment by CineEncoder** [comment]
No, the preset file is still stored in the same place, the indexing of parameters in the preset file matters, when I add a new parameter, the indexing changes. Starting from version 3.5.3, there are no plans to add new parameters or codecs for quite a long time, but if, for example, in six months or a year I add some new AV1 codec, then, accordingly, the preset file will be incompatible with the old versions of the program. And in new versions it will be compatible until I add something new again.
